pom.xml

```diff
@@ -53,7 +53,7 @@
     <properties>
         <apache.directory.server.version>2.0.0-M22</apache.directory.server.version>
         <apache.directory.api.version>1.0.0-M33</apache.directory.api.version>
-        <avro.version>1.7.7</avro.version>
+        <avro.version>1.8.0</avro.version>
```

Comment:
It looks like 1.8.2 is available, any reason not to update to the most recent version? Even if not that, 1.8.1 has been available for quite awhile.